tests/lib/services/alexa/request/lifecycle/response.unit.ts

```diff
@@ -18,13 +20,19 @@ describe('response lifecycle unit tests', () => {
     const turnGet = sinon.stub();
     turnGet.onFirstCall().returns(null);
     turnGet.onSecondCall().returns(true);
-
+    const config = {} as Config;
     const runtime = {
       storage: { get: storageGet },
       turn: { get: turnGet, set: sinon.stub() },
       stack: { isEmpty: sinon.stub().returns(true) },
       variables: { get: sinon.stub().returns(false) },
       getFinalState: sinon.stub().returns(finalState),
+      services: {
+        analyticsClient: AnalyticsClient(config),
+      },
+      getVersionID: () => {
+        return '';
+      },
```

Comment:
same as before this needs to be a stub. 
```suggestion
      getVersionID: sinon.stub().returns(VERSION_ID),
```

Reply:
done